lib/msal-browser/docs/events.md

```diff
@@ -0,0 +1,48 @@
+# Events
+
+MSAL.js 2.0 provides event APIs that are available to users of our core library and wrapper libraries. These events are related to auth and what MSAL is doing, and can be used in applications to update UI, show error messages, and so on.
```

Comment:
2.x? or maybe Msal Browser since we may have other major versions of this

Reply:
"MSAL.js v2"  maybe?

Reply:
2.4+?

Reply:
I'd vote for "MSAL Browser starting at version 2.4"

Reply:
don't mean to add to the chaos here, but I would go with package name `@azure/msal-browser`. I think it's the easiest way to distinguish between the different code bases we maintain.

Reply:
Thanks all for the suggestions! I have changed it to incorporate mostly everything. Let me know if you strongly disagree with the change/have more suggestions.